distributor-api-v1/use-cases/on-session-payments.md

```diff
@@ -0,0 +1,30 @@
+# On session payments
+
+Whenever a user create reservation there is an option to allow him pay directly. This needs to be correctly configured via
+settlement rules on `RateGroup`. We provide this functionality only for `Rates` where the payment should be automatically
+taken immediately when the reservation group is created. In order to simplify the workflow we offer this functionality
+in our payment gateway via a feature called "Payment request".
```

Comment:
This paragraph is really similar to the one in `distributor-api-v1/use-cases/README.md`. Could some parts be unnecessarily duplicated and could some be moved to just one place?